pkg/api/types.go

```diff
@@ -71,6 +71,8 @@ const (
 	Creating ProvisioningState = "Creating"
 	// Updating means the existing OSA resource is being updated.
 	Updating ProvisioningState = "Updating"
+	// AdminUpdating means the existing OSA resource is being updated with admin privileges.
+	AdminUpdating ProvisioningState = "AdminUpdating"
```

Comment:
What is the difference here now between Updating and AdminUpdating? 
What about this: https://github.com/openshift/openshift-azure/blob/master/pkg/api/admin/api/types.go#L89 

In addition we have validation tests: 
https://github.com/openshift/openshift-azure/blob/e3a61726057308e5bd471fc9d4918eccd819647c/pkg/api/validate/validators.go#L312 
We have validation for this, which need to be updated.

Overall where do these states is being used apart fakeRP? How it being consumed?

Reply:
`AdminUpdating` will be set when an update (PUT) to the OSA resource is initiated via the admin API. `Updating` will be set when the update is unprivileged.